doc/api.md

```diff
@@ -0,0 +1,47 @@
+# API
+
+## travelnet.register_travelnet_box
+
+Lets you register your own travelnet boxes with a custom color, name and dye ingredient
+
+Example for a pink travelnet:
+```lua
+travelnet.register_travelnet_box({
+	nodename = "travelnet:travelnet_pink",
+	color = "^[multiply:#FFC0CB",
+	dye = "pink"
```

Comment:
IMO it would make more sense to reduce this to just the color code, and have the `^[multiply:` bit in the registering code. Also maybe have the dye be an item, for more flexibility...

```suggestion
	color = "#FFC0CB",
	dye = "dye:pink"
```